__tests__/api/profiles/get-profile-public.test.js

```diff
@@ -23,7 +25,7 @@ const arrayOfDetailUser = expect.objectContaining({
 describe('GET /profiles/get-profile-public/:username', () => {
   test('should return 200 OK with detail user found', async () => {
     // Execution
-    const response = await request(app).get('/profiles/get-profile-public/anon_username_0');
+    const response = await request(app).get(`/profiles/get-profile-public/${anonymousUsername}`);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change also improves the test's flexibility by using the dynamically generated anonymous username instead of a hardcoded one. This ensures that the test will still pass even if the way anonymous usernames are generated changes in the future.
```diff
-    const response = await request(app).get('/profiles/get-profile-public/anon_username_0');
+    const response = await request(app).get(`/profiles/get-profile-public/${anonymousUsername}`);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->